js2-mode.el

```diff
@@ -226,6 +226,13 @@ variable with predicate PRED."
   (make-variable-buffer-local name)
   (put name 'safe-local-variable pred))
 
+(defcustom js2-ignored-warnings nil
+  "A list of warning message types that will not be reported.
+
+Possible values are the keys of `js2-message-table'."
+  :group 'js2-mode
+  :type 'list)
```

Comment:
I'm not familiar enough with `defcustom` types to comment on the "maybe", I'lll have to defer to you.